ADOPTERS.md

```diff
@@ -11,6 +11,7 @@ be added to this list as they transition to production deployments.
 
 <a href="https://www.jd.com" border="0" target="_blank"><img alt="JD.com" src="https://raw.githubusercontent.com/goharbor/website/main/docs/img/adopters/jd.png" height="50"></a>&nbsp; &nbsp; &nbsp;
 <a href="https://www.trendmicro.com" border="0" target="_blank"><img alt="trendmicro" src="https://raw.githubusercontent.com/goharbor/website/main/docs/img/adopters/trendmicro.png" height="50"></a>&nbsp; &nbsp; &nbsp; &nbsp;
+<a href="https://pitsdatenrettung.de/" target="_blank" border="0"><img alt="PITS Globale Datenrettungsdienste" src="https://raw.githubusercontent.com/goharbor/website/main/docs/img/adopters/pits-globale-datenrettungsdienste.svg" height="50"></a>&nbsp; &nbsp; &nbsp; &nbsp;
```

Comment:
Currently this dependent PR is merged, but the `src` URL(https://raw.githubusercontent.com/goharbor/website/main/docs/img/adopters/pits-globale-datenrettungsdienste.svg) is still 404; Maybe the URL should be changed to:
https://raw.githubusercontent.com/goharbor/website/main/docs/img/adopters/pits-globale-datenrettungsdienste.png